<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleWindow</class>
 <widget class="QDialog" name="BattleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Морской бой</string>
  </property>
  <widget class="QLabel" name="labelPlayer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ваше поле</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEnemy">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Поле противника</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutPlayer" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutEnemy" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReplay">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сыграть снова</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayerShips">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ваши корабли: 10</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEnemyShips">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Корабли противника: 10</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
